clients/src/main/resources/common/message/EndQuorumEpochRequest.json

```diff
@@ -18,26 +18,41 @@
   "type": "request",
   "listeners": ["controller"],
   "name": "EndQuorumEpochRequest",
-  "validVersions": "0",
-  "flexibleVersions": "none",
+  "validVersions": "0-1",
```

Comment:
Please add a comment to mention what changed in v.1.

Reply:
Fixed.